File: Python/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop):
    def __init__(self, id=None, nama=None, harga=None, stok=None, jenis=None, bahan=None, warna=None):
        super().__init__(id, nama, harga, stok)
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna

    # setter dan getter

    # setter jenis
    def set_jenis(self, jenis):
        self.jenis = jenis

    # setter bahan
    def set_bahan(self, bahan):
        self.bahan = bahan

    # setter warna
    def set_warna(self, warna):
        self.warna = warna

    # getter jenis
    def get_jenis(self):
        return self.jenis

    # getter bahan
    def get_bahan(self):
        return self.bahan

    # getter warna
    def get_warna(self):
        return self.warna

File: Java/Aksesoris.java
public class Aksesoris extends PetShop{
    private String jenis;
    private String bahan;
    private String warna;

    public Aksesoris(){
    }

    public Aksesoris(int id, String nama, double harga, int stok, String jenis, String bahan, String warna){
        super(id, nama, harga, stok);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    public void setJenis(String jenis){
        this.jenis = jenis;
    }

    public void setBahan(String bahan){
        this.bahan = bahan;
    }

    public void setWarna(String warna){
        this.warna = warna;
    }

    public String getJenis(){
        return jenis;
    }

    public String getBahan(){
        return bahan;
    }

    public String getWarna(){
        return warna;
    }

}